### PR TITLE
Clarification of the customer's reports (Order Total, New, Order Count, Wish Lists, Segments)

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describes clarification of the customer's reports:
- Order Total
- New
-  Order Count
-  Wish Lists
-  Segments

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/reports/customer-reports.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=

### CODE REVIEW

File: src/reports/customer-reports.md [hunk 1]
```diff
@@ -31,6 +51,16 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **New**.
...
 
+### Workspace Controls
+
+|Control|Description|
+|--- |--- |
+|FromTo| Search by date range. Initiates a search for the accounts based on the start to end date.|
+|Show By|defines the granularity of the order record splitting. Options: Month/ Day/ Year |
+|Refresh|Updates the grid to the specified filters.|
+|Export|Exports the selected records as a CSV or Excel XML file.|
+|Scope| Displayed in which store the report will be generated.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:62]:
```suggestion
|Scope|Used to set the site or store for which the report will be generated.|
```

File: src/reports/customer-reports.md [hunk 2]
```diff
@@ -31,6 +51,16 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **New**.
 
 The New Accounts Report shows the number of new customer accounts opened during a specified time interval or date range.
 
+### Workspace Controls
+
+|Control|Description|
+|--- |--- |
+|FromTo| Search by date range. Initiates a search for the accounts based on the start to end date.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:58]:
```suggestion
|From / To|Used to define a search for the new accounts based on the start and end date.|
```

File: src/reports/customer-reports.md [hunk 3]
```diff
@@ -31,6 +51,16 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **New**.
...
+|Show By|defines the granularity of the order record splitting. Options: Month/ Day/ Year |
```
Comment (leslietilling) [inline src/reports/customer-reports.md:59]:
```suggestion
|Show By|Defines the granularity of the order record splitting. Options: Month / Day / Year |
```

File: src/reports/customer-reports.md [hunk 4]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
 
 The Customer Wish List Report provides information about customers Wish Lists.
 
+### Workspace Controls
+
+|Control|Description|
+|--- |--- |
+|Scope| Displayed in which store the report will be generated.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:78]:
```suggestion
|Scope|Used to set the site or store for which the report will be generated.|
```

File: src/reports/customer-reports.md [hunk 5]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
 
 The Customer Wish List Report provides information about customers Wish Lists.
 
+### Workspace Controls
```
Comment (leslietilling) [inline src/reports/customer-reports.md:74]:
```suggestion
### Workspace controls
```

File: src/reports/customer-reports.md [hunk 6]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
 
 The Customer Wish List Report provides information about customers Wish Lists.
```
Comment (leslietilling) [inline src/reports/customer-reports.md:72]:
```suggestion
The Customer Wish List Report provides information about customer Wish Lists.
```

File: src/reports/customer-reports.md [hunk 7]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+### Workspace Controls
+
+|Control|Description|
+|--- |--- |
+|Scope| Displayed in which store the report will be generated.|
+|Search| Initiates a search by the specified parameters.|
+|Reset Filter| Initiates a reset of all search parameters.|
+|Per Page| Displays how many records are listed at the same time.|
+|Export|Exports the selected records as a CSV or Excel XML file.|
+|FromTo|Search by date range. Initiates a search for the wishlists based on the start to end date.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:83]:
```suggestion
|From / To|Used to define a search for the wish lists based on the start and end date.|
```

File: src/reports/customer-reports.md [hunk 8]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|Wishlist| Initiates a wishlist search by name.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:84]: Sadly, the UI is inconsistent with wish list as one or two words.

```suggestion
|Wishlist| Initiates a wish list search by name.|
```

File: src/reports/customer-reports.md [hunk 9]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|Status| The current status in which the wishlist is displayed. Options: Privet/ Public |
```
Comment (leslietilling) [inline src/reports/customer-reports.md:85]:
```suggestion
|Status| The current status of the wish list. Options: Private / Public |
```

File: src/reports/customer-reports.md [hunk 10]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|Comment| Initiates a search by words in the wishlist comment.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:86]:
```suggestion
|Comment| Initiates a search by text in the wish list comments.|
```

File: src/reports/customer-reports.md [hunk 11]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+
+### Column description
```
Comment (leslietilling) [inline src/reports/customer-reports.md:88]:
```suggestion
### Column descriptions
```

File: src/reports/customer-reports.md [hunk 12]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+
+|Column|Description|
+|--- |--- |
+|Added| Date of the wishlist creation.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:92]:
```suggestion
|Added| Date the wish list was created.|
```

File: src/reports/customer-reports.md [hunk 13]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|Customer| First and last name of the customer, who has created the wishlist.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:93]:
```suggestion
|Customer| First and last name of the customer who created the wish list.|
```

File: src/reports/customer-reports.md [hunk 14]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|Wishlist| Name of the wishlist.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:94]:
```suggestion
|Wishlist| Name of the wish list.|
```

File: src/reports/customer-reports.md [hunk 15]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|Status| The current status in which the wishlist is displayed. Options: Privet/ Public |
```
Comment (leslietilling) [inline src/reports/customer-reports.md:95]:
```suggestion
|Status| The current status of the wish list. Options: Private / Public |
```

File: src/reports/customer-reports.md [hunk 16]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|Product| Name of the product added to wishlist.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:96]:
```suggestion
|Product| Name of the product added to the wish list.|
```

File: src/reports/customer-reports.md [hunk 17]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|SKU| Sku of the product added to wishlist.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:97]:
```suggestion
|SKU| SKU of the product added to the wish list.|
```

File: src/reports/customer-reports.md [hunk 18]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
+|Comment| Сomment described upon wishlist creation.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:98]:
```suggestion
|Comment| Сomment entered when the wish list was created.|
```

File: src/reports/customer-reports.md [hunk 19]
```diff
@@ -50,3 +106,12 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Segments**.
...
 
 The Customer Segment Report provides information about the number of customers in each segment.
+
+### Workspace Controls
+
+|Control|Description|
+|--- |--- |
+|Search| Initiates a search by the specified parameters.|
+|Reset Filter| Initiates a reset of all search parameters.|
+|Action| Initiates the display of segments by parameters. Options: Action/ View Combined Report|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:116]:
```suggestion
|Action| Initiates the display of segments by parameters. Options: Action / View Combined Report|
```

File: src/reports/customer-reports.md [hunk 20]
```diff
@@ -41,6 +71,32 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Wish Lists**.
...
 The Customer Wish List Report provides information about customers Wish Lists.
 
+### Workspace Controls
+
+|Control|Description|
+|--- |--- |
+|Scope| Displayed in which store the report will be generated.|
+|Search| Initiates a search by the specified parameters.|
+|Reset Filter| Initiates a reset of all search parameters.|
+|Per Page| Displays how many records are listed at the same time.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:81]:
```suggestion
|Per Page| Sets the number of records displayed in a single page. |
```

File: src/reports/customer-reports.md [hunk 21]
```diff
@@ -50,3 +106,12 @@ On the _Admin_ sidebar, go to **Reports** > _Customers_ > **Segments**.
...
 The Customer Segment Report provides information about the number of customers in each segment.
+
+### Workspace Controls
+
+|Control|Description|
+|--- |--- |
+|Search| Initiates a search by the specified parameters.|
+|Reset Filter| Initiates a reset of all search parameters.|
+|Action| Initiates the display of segments by parameters. Options: Action/ View Combined Report|
+|Per Page| Displays how many records are listed at the same time.|
```
Comment (leslietilling) [inline src/reports/customer-reports.md:117]: |Per Page| Sets the number of records displayed in a single page. |

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you a lot for reviewing! 

I have implemented all suggestions, could you please approve? 

Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@leslietilling, an error occurred during the Pull Request import.

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@leslietilling, an error occurred during the Pull Request import.

**Comment by magento-engcom-team** [comment]
@leslietilling, an error occurred during the Pull Request import.

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1221`.

**Pull request merged**